README.md

```diff
@@ -1,26 +1,109 @@
-# README
```

Comment:
Your _ReadMe_ needs some improvements so as to make it appear more professional. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this project,  which will likely feature on your portfolio. Kindly ensure to improve upon these aspects of your ReadMe. 
- [ ] Project description, how to set up, run and test this project should be detailed, clear, and well organized.
- [ ] The code aspects in each procedure should be in backticks`.
- [ ] All typos should be corrected. You can install [Grammarly plugin](https://www.grammarly.com/) app to assist you to identify errors in your code.

Reply:
This issue has been addressed well. Great job! :clap: :clap: